README.md

```diff
@@ -15,6 +15,23 @@ This repository contains both L1 and L2 zkSync smart contracts. For their descri
 It is used as a submodule of a private repo. Compilation and test scripts should work without additional tooling, but
 others may not.
 
+## Testing
```

Comment:
I guess it's only about system contracts, should this be placed in the global readme? Actually, this information is already mentioned in the system contracts readme